project/src/js/lichess/friends.js

```diff
@@ -1,8 +1,40 @@
-import { caseInsensitiveSort } from '../utils';
-
 var onlineFriends = [];
 
+var makeUser = function(name, isPlaying) {
```

Comment:
Prefer the ```function makeUser(name, isPlaying) {}``` notation (named function) here; same comment for the 3 other functions below